crates/conjure_core/src/ast/expressions.rs

```diff
@@ -156,7 +155,7 @@ fn range_vec_bounds_i32(ranges: &Vec<Range<i32>>) -> (i32, i32) {
                 }
             }
             Range::Bounded(i, j) => {
-                if *i < min {}
+                *i < min;
```

Comment:
this line looks a bit pointless now, what's going on here?

Reply:
I'm not sure what this function is meant to do

Reply:
Do we want to find the intersection of the ranges or the union?